pages/lib/extract_df.py

```diff
@@ -407,11 +426,11 @@ def enthalpy(df, name):
 
 
 def convert_data(df, mapping_json):
-    df[ColNames.ADAPTIVE_COMFORT] = df[ColNames.ADAPTIVE_COMFORT] * 1.8 + 32
-    df[ColNames.ADAPTIVE_CMF_80_LOW] = df[ColNames.ADAPTIVE_CMF_80_LOW] * 1.8 + 32
-    df[ColNames.ADAPTIVE_CMF_80_UP] = df[ColNames.ADAPTIVE_CMF_80_UP] * 1.8 + 32
-    df[ColNames.ADAPTIVE_CMF_90_LOW] = df[ColNames.ADAPTIVE_CMF_90_LOW] * 1.8 + 32
-    df[ColNames.ADAPTIVE_CMF_90_UP] = df[ColNames.ADAPTIVE_CMF_90_UP] * 1.8 + 32
+    temperature(df, ColNames.ADAPTIVE_COMFORT)
```

Comment:
the name of the function should be more meaningful. it should be `convert_t_to_f`